clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java

```diff
@@ -679,4 +683,23 @@ private HeartbeatRequestManager createHeartbeatRequestManager() {
             heartbeatRequestState,
             backgroundEventHandler);
     }
+
+    private HeartbeatRequestManager createHeartbeatRequestManager(
```

Comment:
turned out we are using testBuilder in this test again.  So we might want to refactor the test and turn everything to mock.